datasets/noaa/ghcnd_countries/ghcnd_countries_dag.py

```diff
@@ -33,9 +34,9 @@
 ) as dag:
 
     # Run CSV transform within kubernetes pod
-    transform_csv = kubernetes_pod_operator.KubernetesPodOperator(
+    transform_csv = kubernetes_pod.KubernetesPodOperator(
         task_id="transform_csv",
-        name="lightning_strikes_by_year",
+        name="ghcnd_countries",
```

Comment:
Same here, there's some weird renaming that shouldn't be applied

Reply:
@adlersantos I think that what the history is telling us is that it at one point in time the dag file refered to the wrong pipeline, but that has since been resolved.  Thanks! Nick.